### DISCUSSION TITLE
How to combine two or more script into one using json-ld

### DISCUSSION THREAD

**Discussion by kasaviva** [json-ld/json-ld.org#828] [Q&A]
Hello, i have a few scripts and world like to combine them into one, but don't know how and can't find a tool to do it. 
thanks in advanced.

**Comment by BigBlueHat** [comment]
Hi @kasaviva,

Can you explain more about your use case? Or provide example code of what you're trying to do?

When you say "script" are you referring to JavaScript code?

Thanks!
🎩
